tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -338,6 +338,44 @@ async def test_get_balance(wallet_rpc_environment: WalletRpcTestEnvironment):
     await assert_get_balance(wallet_rpc_client, wallet_node, cat_wallet)
 
 
+@pytest.mark.asyncio
+async def test_get_farmed_amount(wallet_rpc_environment: WalletRpcTestEnvironment):
+    env = wallet_rpc_environment
+    wallet: Wallet = env.wallet_1.wallet
+    full_node_api: FullNodeSimulator = env.full_node.api
+    wallet_rpc_client = env.wallet_1.rpc_client
+    wallet_node: WalletNode = env.wallet_1.node
+    await full_node_api.farm_blocks_to_wallet(2, wallet)
+
+    result = await wallet_rpc_client.get_farmed_amount()
+
+    expected_result = {
+        "farmed_amount": 4000000000000,
+        "farmer_reward_amount": 500000000000,
+        "fee_amount": 0,
+        "last_height_farmed": 2,
+        "pool_reward_amount": 3500000000000,
+        "success": True,
+    }
+
+    assert result == expected_result
+
+    our_ph = await wallet.get_new_puzzlehash()
+    tx = await wallet.generate_signed_transaction(
+        amount=uint64(5),
+        puzzle_hash=our_ph,
+        fee=uint64(100),
+    )
+    await wallet.push_transaction(tx)
```

Comment:
I don't think we care where this send goes?

Using `fee_amount` explicitly documents the relationship between the fee here and the fee amount below.

```suggestion
    fee_amount = 100
    tx = await wallet.generate_signed_transaction(
        amount=uint64(5),
        puzzle_hash=bytes32([0] * 32),
        fee=uint64(fee_amount),
    )
    await wallet.push_transaction(tx)
```